src/app/app.component.html

```diff
@@ -1,3 +1,4 @@
 
 <router-outlet></router-outlet>
-<ngx-spinner bdColor = "rgba(0,0,0,0.5)" size = "medium" color = "#fff" type = "pacman" [fullScreen] = "true"><p style="color: white" > Working... </p></ngx-spinner>
\ No newline at end of file
+<app-loader ></app-loader>
+<ngx-spinner bdColor = "rgba(0,0,0,0.5)" size = "medium" color = "#fff" type = "pacman" [fullScreen] = "true"><p style="color: white" > Working... </p></ngx-spinner>
```

Comment:
Need to remove ngx-spinner once all the functionality has been ported to app-loader